templates/url_redirect/index.html.twig

```diff
@@ -3,30 +3,41 @@
 {% block title %}{% trans %}Redirected Urls {% endtrans %}{% endblock %}
 
 {% block body %}
+    {% if app.user %}
     <h1>{% trans %}All your redirected urls{% endtrans %}</h1>
 
     <table class="table">
         <thead>
             <tr>
                 <th>Id</th>
                 <th>{% trans %}Your Url {% endtrans %}</th>
+                <th>{% trans %}Description {% endtrans %}</th>
                 <th>{% trans %}Generated Url{% endtrans %}</th>
+                <th>{% trans %}Actions{% endtrans %}</th>
             </tr>
         </thead>
         <tbody>
         {% for url_redirect in url_redirect %}
```

Comment:
element tablicy nazywa się tak samo jak sama tablica. To będzie powodowało problemy